### ISSUE TITLE
Uncaught ReferenceError when running code cell

### ISSUE THREAD

**Issue by dnowacki-usgs** [nteract/hydrogen#401]
Attempting to run a cell of a Python 2.7 script results in the following error, the full stack trace of which is below. Running code in non-cell mode still works fine. I'm prompted to check for a package update, but I'm already running the latest version. I tried uninstalling and reinstalling hydrogen with the same result. Any suggestions?

**Atom Version**: 1.9.7
**System**: Mac OS X 10.10.5
**Thrown From**: [hydrogen](https://github.com/nteract/hydrogen) package, vundefined

### Stack Trace

Uncaught ReferenceError: _ is not defined

```
At /Users/dnowacki/.atom/packages/hydrogen/lib/code-manager.coffee:160

ReferenceError: _ is not defined
    at CodeManager.module.exports.CodeManager.isComment (/Users/dnowacki/.atom/packages/hydrogen/lib/code-manager.coffee:160:16)
    at CodeManager.module.exports.CodeManager.getCurrentCell (/Users/dnowacki/.atom/packages/hydrogen/lib/code-manager.coffee:104:41)
    at Object.module.exports.Hydrogen.runCell (/Users/dnowacki/.atom/packages/hydrogen/lib/main.coffee:324:37)
    at atom-text-editor.subscriptions.add.atom.commands.add.hydrogen:run-cell (/Users/dnowacki/.atom/packages/hydrogen/lib/main.coffee:54:38)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/command-palette/lib/command-palette-view.js:183:32)
    at CommandPaletteView.module.exports.SelectListView.confirmSelection (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
    at space-pen-div.atom.commands.add.core:confirm (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:98:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
     -1:21.8.0 hydrogen:run-cell (atom-text-editor.editor)
  2x -1:09.3.0 hydrogen:run-and-move-down (atom-text-editor.editor.is-focused)
     -0:59.6.0 editor:toggle-line-comments (atom-text-editor.editor.is-focused)
     -0:59.3.0 core:move-up (atom-text-editor.editor.is-focused)
     -0:58.9.0 editor:toggle-line-comments (atom-text-editor.editor.is-focused)
  3x -0:58.1.0 core:move-down (atom-text-editor.editor.is-focused)
     -0:57.3.0 editor:toggle-line-comments (atom-text-editor.editor.is-focused)
     -0:53.5.0 hydrogen:run-and-move-down (atom-text-editor.editor.is-focused)
     -0:45.1.0 core:move-down (atom-text-editor.editor.is-focused)
  2x -0:44.6.0 core:select-up (atom-text-editor.editor.is-focused)
     -0:43.8.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:40.8.0 hydrogen:run-and-move-down (atom-text-editor.editor.is-focused)
     -0:37.5.0 hydrogen:run-cell (atom-text-editor.editor.is-focused)
     -0:02.1.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:00.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:00.0 hydrogen:run-cell (atom-text-editor.editor)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
Hydrogen, v0.12.1 (active)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.51.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.43.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.3.2 (active)
one-light-ui, v1.3.2 (inactive)
one-dark-syntax, v1.2.0 (active)
one-light-syntax, v1.2.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.2 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.0 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.0 (active)
bookmarks, v0.41.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (active)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.39.0 (active)
fuzzy-finder, v1.3.0 (active)
git-diff, v1.1.0 (active)
find-and-replace, v0.198.0 (inactive)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (active)
image-view, v0.58.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (active)
metrics, v0.53.1 (active)
notifications, v0.64.1 (active)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.238.0 (active)
snippets, v1.0.2 (active)
spell-check, v0.67.1 (active)
status-bar, v1.4.1 (active)
styleguide, v0.46.0 (active)
symbols-view, v0.113.0 (inactive)
tabs, v0.98.0 (active)
timecop, v0.33.1 (active)
tree-view, v0.208.0 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.34.0 (active)
whitespace, v0.32.2 (active)
wrap-guide, v0.38.1 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (active)
language-coffee-script, v0.47.0 (active)
language-csharp, v0.12.1 (active)
language-css, v0.36.2 (active)
language-gfm, v0.86.0 (active)
language-git, v0.13.0 (active)
language-go, v0.42.0 (active)
language-html, v0.44.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.22.0 (active)
language-javascript, v0.119.0 (active)
language-json, v0.18.0 (active)
language-less, v0.29.3 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.0 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (active)
language-ruby, v0.68.5 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.52.0 (active)
language-shellscript, v0.22.3 (active)
language-source, v0.9.0 (active)
language-sql, v0.21.1 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.8 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by lgeiger** [comment]
Thanks for reporting. We just released `v0.12.2` to fix this.

**Comment by dnowacki-usgs** [comment]
Now that's service! Thanks.
